x-pack/plugins/observability/server/ui_settings.ts

```diff
@@ -225,11 +216,9 @@ export const uiSettings: Record<string, UiSettings> = {
     }),
     value: false,
     description: i18n.translate('xpack.observability.enableInfrastructureHostsViewDescription', {
-      defaultMessage:
-        '{technicalPreviewLabel} Enable the Hosts view in the Infrastructure app. {feedbackLink}.',
+      defaultMessage: '{betaLabel} Enable the Hosts view in the Infrastructure app.',
```

Comment:
Is this change from `technicalPreviewLabel` to `betaLabel` intentional? What is the difference?